libopae/src/bitstream.c

```diff
@@ -552,7 +552,7 @@ fpga_result read_gbs_metadata(const uint8_t *bitstream,
 				e = memcpy_s(gbs_metadata->afu_image.afu_clusters.name,
 						AFU_NAME_LEN,
 						json_object_get_string(name),
-						sizeof(json_object_get_string(name)));
+						strnlen_s(json_object_get_string(name), AFU_NAME_LEN));
```

Comment:
Would it better to use `json_object_get_string_len`?
I'm not sure which is better.